File: csharp/MapleStory.Test/GuildAPI.cs
using NUnit.Framework;
using MapleStory.OpenAPI;

namespace MapleStory.Test
{
    [TestFixture]
    public class GuildAPI
    {
        private static readonly string apiKey = Environment.GetEnvironmentVariable("API_KEY")!; // Your API Key
        private static readonly MapleStoryAPI api = new MapleStoryAPI(apiKey);
        private static readonly string ogid = "f5992063d46d6f8718e514fca5428960";

        [Test]
        public async Task GetGuild()
        {
            var guildName = "붕붕";
            var worldName = "크로아";
            var response = await api.GetGuild(guildName, worldName);
            Console.WriteLine(response.ToJson());
        }

        [Test]
        public async Task GetGuildBasic()
        {
            var response = await api.GetGuildBasic(ogid);
            Console.WriteLine(response.ToJson());
        }
    }
}

File: java/src/test/java/GuildApi.java
import dev.spiralmoon.maplestory.api.MapleStoryApi;
import dev.spiralmoon.maplestory.api.dto.guild.*;
import org.junit.jupiter.api.DisplayName;
import org.junit.jupiter.api.Test;

import java.io.IOException;

@DisplayName("길드 정보 조회")
public class GuildApi {

    private static final String apiKey = System.getProperty("API_KEY"); // Your API Key
    private static final MapleStoryApi api = new MapleStoryApi(apiKey);
    private static final String ogid = "f5992063d46d6f8718e514fca5428960";

    @Test
    @DisplayName("길드 식별자(oguild_id) 정보 조회")
    void getGuild() throws IOException {
        String guildName = "붕붕";
        String worldName = "크로아";
        GuildDTO response = api.getGuild(guildName, worldName).join();
        System.out.println(response.toString());
    }

    @Test
    @DisplayName("기본 정보 조회")
    void getGuildBasic() throws IOException {
        GuildBasicDTO response = api.getGuildBasic(ogid).join();
        System.out.println(response.toString());
    }
}
